test/e2e/next-font/with-proxy.test.ts

```diff
@@ -0,0 +1,45 @@
+import { createNextDescribe, FileRef } from 'e2e-utils'
+import { check } from 'next-test-utils'
+import { join } from 'path'
+
+createNextDescribe(
+  'next/font/google with proxy',
+  {
+    skipStart: true,
+    files: new FileRef(join(__dirname, 'with-proxy')),
+    dependencies: {
+      react: 'latest',
+      'react-dom': 'latest',
+      'https-proxy-agent': '5.0.1',
```

Comment:
Should we pre-compile this and re-use it elsewhere we are fetching external resources?